tce/src/applibs/LLVMBackend/plugin/TCEISelLowering.cc

```diff
@@ -702,15 +702,18 @@ TCETargetLowering::TCETargetLowering(
                       << "lots of 8-bit loads." << std::endl;
         }
         setLoadExtAction(ISD::EXTLOAD, MVT::i32, MVT::i8, Custom);
-        setLoadExtAction(ISD::SEXTLOAD, MVT::i32, MVT::i8, Custom);
```

Comment:
You do not set anymore the custom selector for the sign extending loads. If we do have native sign extending loads, does it still go to the Custom selector by default and work like it should?